test/unittests/NDB_BVL_Instrument_Test.php

```diff
@@ -1624,7 +1624,7 @@ function testDiff()
             $this->_instrument->diff($otherInstrument),
             [
                 [
-                    'TableName'      => 'medical_history',
+                    'TestName'       => 'medical_history',
```

Comment:
Please change 'medical_history' to 'Test' after "php/libraries/NDB_BVL_Instrument.class.inc" diff function  changed.